.github/workflows/release.yaml

```diff
@@ -158,3 +158,14 @@ jobs:
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_TOOLBOX_WEBHOOK_URL }}
         if: ${{ success() }}
+  
+  homebrew:
```

Comment:
actually, if we add this as a separate post-release workflow (using I think `on: release`) , then we can safely re-run this without needing to worry about the release workflow itself. Thoughts?

Reply:
I updated the branch to a separate post-release workflow (I have not run this workflow though)